### DISCUSSION TITLE
Title da pagina puxando do banco de dados

### DISCUSSION THREAD

**Discussion by oalannn** [frontendbr/forum#1567] [Perguntas e Respostas]
Olá sou novo aqui e na área... , estou com um projeto de um website de filmes, e gostaria de saber como faço para que na pagina do filme "X" apareça no <title>"X"</title> o nome do filme selecionado, técnica de SEO, tentei incluir um `<title><?php echo $exibe->titulo;?></title>` , mas não deu certo....

**Comment by theandersonn** [comment]
@oalannn Sua mensagem ficou longa e a estrutura ficou quebrada, não dá pra entender o código.

Separa as declarações de css num arquivo a parte.

Esse title está dentro do head?

Debuga o retorno do php.
